test/e2e/upgrade/alert/alert.go

```diff
@@ -91,7 +92,8 @@ func (t *UpgradeTest) Test(f *framework.Framework, done <-chan struct{}, upgrade
 		criticalAlertQuery: false,
 	}
 
-	helper.RunQueries(tests, t.oc, ns, execPod.Name, t.url, t.bearerToken)
+	err := helper.RunQueries(tests, t.oc, ns, execPod.Name, t.url, t.bearerToken)
+	o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
Why this btw?

Reply:
because `RunQueries` had `because `o.Expect(err).NotTo(o.HaveOccurred())` inside and now it returns `error`, so storage metric tests can mark themselves as flaky instead of failed.